### ISSUE TITLE
[PNI] Update Related Articles Section on Article Pages

### ISSUE THREAD

**Issue by beccaklam** [mozilla/foundation.mozilla.org#9921]
Related to #9664

We are creating a new related articles block (#9874) to address some issues rasied in #9664 and therefore will be making some updates to the Related Articles section on Article pages. This work will also unblock getting related articles onto article pages.

**Acceptance Criteria**
- [ ] As an editor, I can only populate three articles in the related articles section 
- [ ] As an editor, I see the related articles section at the bottom of the articles cms form/page
- [ ] As a visitor, I no longer see the "primary styling" attached to any content I enter into the related articles section. All content now only affects the "secondary styling"/bottom related articles section.
- [ ] As a visitor, I see the correct amount of spacing (48px) between the newsletter signup box and the related articles section at the bottom of the article page as displayed in the [figma design](https://www.figma.com/file/4QFUwtwPN88j74no526S3y/Privacy-Not-Included-Master-File?node-id=3%3A8120&t=9cQNewxeM8hsAYwv-1)

**Comment by tbrlpld** [comment]
@beccaklam @nancyt1 Since we are removing the concept of primary/secondary from the article page, I would also propose to change the `id` of the "what to read next" section that was added for GTM tracking. I would propose to use the a data attribute `data-gtm-article-what-to-read-next`. Does that work for you?

**Comment by nancyt1** [comment]
@tbrlpld Can we shorten it to something like `data-gtm-article-related-reading` since we were thinking the new streamfield block would be called `data-gtm-inline-related-reading` or something like that?

**Comment by tbrlpld** [comment]
@nancyt1 Is there a particular reason why you want to keep this short? I am wondering if this might cause confusion by using these different terms for the same list. But in the end it does not matter much to me, just wondering. 

I am now also thinking using values for a `data-gtm` attribute makes more sense then my initial proposal. So `data-gtm="article-what-to-read-next"` or `data-gtm="article-related-reading"`.

**Comment by nancyt1** [comment]
@tbrlpld I guess short wasn't the main reason, I think it would be better to name it based on it's function and not tie it too closely to the title copy since I feel like that's something that could easily change but the attribute should still make sense.

**Comment by tbrlpld** [comment]
@nancyt1 That makes sense. How about `related-articles` then? That is the name of the field we store the data and it not tied to the frontend copy?

**Comment by nancyt1** [comment]
@tbrlpld I like that! I think we can assume based on URL but I'm hesitant to use just `related-articles` unless we update all the other related article blocks. Is that easy to do? It would help streamline our tagging.

**Comment by tbrlpld** [comment]
@nancyt1 So `article-related-articles` then?

**Comment by tbrlpld** [comment]
@beccaklam This is live on staging now. 

* https://foundation.mofostaging.net/cms/pages/20974/edit/ 
* https://foundation.mofostaging.net/en/privacynotincluded/articles/alexa-help-me-which-smart-speaker-has-the-best-privacy/

**Comment by beccaklam** [comment]
Yay! This is working and looking great, thank you @tbrlpld!

**Comment by nancyt1** [comment]
@tbrlpld  Actually lets go with `related-articles`! I can make a new ticket to update the other blocks.

**Comment by tbrlpld** [comment]
Thanks for testing @beccaklam

**Comment by tbrlpld** [comment]
Thanks @nancyt1 I will create a quick follow up PR to fix this location.

**Comment by beccaklam** [comment]
@tbrlpld Question, will this update affect related articles on other pages besides the article page? E.g. Product review pages and the article index page

**Comment by tbrlpld** [comment]
@beccaklam Nope, these changes are limited to the article page. Did you see unintended side effects elsewhere?

**Comment by beccaklam** [comment]
@tbrlpld No, I was checking just in case!
